### ISSUE TITLE
Code Release

### ISSUE THREAD

**Issue by abhigoku10** [Xavier-Pan/WSGCN#1]
@Xavier-Pan @johnnylu305  read ur paper and its very interesting can you let us kwn when is the code and pre-trained models releasing

**Comment by johnnylu305** [comment]
@abhigoku10 
Thank you for your interest in our work. We are preparing the code, but it may take some time. One of the reasons is that we use powerful machines with Tesla V100 32GB and 10TB SSD/HDD to train our model. I hope that we can release a version for typical deep learning machines. Hence, I am very sorry that I cannot give you any promise about the date. 

If you are interested in the second stage model, you can refer to https://github.com/johnnylu305/deeplab-imagenet-pytorch. Note that the code is highly based on https://github.com/kazuto1011/deeplab-pytorch. In addition, we will not release the pre-trained weight for "each" GCN. But, I believe you can generate the complete pseudo labels through our code in the future.

**Comment by abhigoku10** [comment]
@johnnylu305  thanks for ur response eagerly awaiting the code release !!! Great work to the authors and the team
